keyboards/hexon38/hexon38.h

```diff
@@ -0,0 +1,20 @@
+// see https://github.com/pepaslabs/hexon38
+
+#ifndef HEXON38_H
+#define HEXON38_H
+
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
We're moving away from `KEYMAP`. Could you change this to `LAYOUT`? 
```suggestion
 #define LAYOUT( \
```